drivers/pcie/host/pcie.c

```diff
@@ -533,6 +540,12 @@ static int pcie_init(void)
 		.flags = PCIE_SCAN_RECURSIVE,
 	};
 
+#ifdef CONFIG_PCIE_PRT
+	if (acpi_legacy_irq_init(
```

Comment:
not sure it's the right place to call this actually, we could just initialize it all in acpi code.

Reply:
We need PCI Bus ACPI HW/PNP ID from DTS for retrieving during init. Later call is only per device which just return cached value.